src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String getRandomColor() {
+
+        String[] colors = {

```

Comment:
These colors are more of a constant so putting them on method level is a bad idea. Additionally, you could extract them to enum.